test/archive.cpp

```diff
@@ -360,10 +360,10 @@ TEST(ZimArchive, illustration)
       const zim::Archive archive(testfile.path);
       ASSERT_TRUE(archive.hasIllustration(48)) << ctx;
       auto illustrationItem = archive.getIllustrationItem(48);
-      if(testfile.category == "nons") {
-        ASSERT_EQ(illustrationItem.getPath(), "Illustration_48x48@1") << ctx;
-      } else {
+      if(testfile.category == "withns") {
         ASSERT_EQ(illustrationItem.getPath(), "I/favicon.png") << ctx;
+      } else {
+        ASSERT_EQ(illustrationItem.getPath(), "Illustration_48x48@1") << ctx;
```

Comment:
This change is a distraction. The PR can be as good without it.

Reply:
This is a distraction if you have only two categories. But with a new category, using the `else` branch, this is important.

Reply:
Indeed. I missed that.